src/main/java/com/derongan/minecraft/deeperworld/DeeperCommandExecutor.kt

```diff
@@ -33,5 +35,31 @@ object DeeperCommandExecutor : IdofrontCommandExecutor() {
                     sender.info("${player.name} is in section ${section.key}")
             }
         }
+        "syncedtime" {
+            "set"{
+                val time by intArg()
+                action {
+                    DeeperConfig.data.time.mainWorld?.let {
+                        it.time = time.toLong()
+                    } ?: return@action
+
+                    DeeperConfig.data.time.syncedWorlds.forEach {
+                        it.key.time = time.toLong() + it.value
+                    }
```

Comment:
If you want this to be saved to the config, you should be able to do `DeeperConfig.queueSave()` or something similar, but that just reserializes the entire structure and I forget if it preserves comments or not... maybe I should figure out something for that 🤔.

You can ignore this for now but I guess just know this won't be persisted between restarts.